tasks/build-preview-urls-comment.js

```diff
@@ -56,7 +56,7 @@ export const buildPreviewURLComment = (ref) => {
     const previewLinks = [];
     const themes = ['Spectrum', 'Express', 'Spectrum-two'];
     const scales = ['Medium', 'Large'];
-    const colors = ['Lightest', 'Light', 'Dark', 'Darkest'];
+    const colors = ['Light', 'Dark'];
```

Comment:
We wanted to keep this to maintain a backward compatibility baseline images for Spectrum 1 and Express

Reply:
We don't test these in VRTs already. The reason for removing them from here was just to avoid getting broken links in the github comment. 
These are deprecated and so even though we support them right now... we don't test them. If you want I can update our configs to once again start testing for these in VRTs and subsequently can revert this change. But we'll have to do one or the other

Reply:
Yes make sense!